ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -16,6 +17,22 @@ public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
         where TFrameworkElement : FrameworkElement
         where TLayoutShadowNode : LayoutShadowNode
     {
+        /// <summary>
+        /// Instantiates the base class <see cref="BaseViewManager{TFrameworkElement, TReactShadowNode}"/>.
+        /// </summary>
+        protected BaseViewManager()
+        {
+        }
+
+        /// <summary>
+        /// Instantiates the base class <see cref="BaseViewManager{TFrameworkElement, TReactShadowNode}"/>.
+        /// </summary>
+        /// <param name="eventDispatcher">The event dispatcher to associate with this instance</param>
+        protected BaseViewManager(IEventDispatcher eventDispatcher)
```

Comment:
The ability to inject the EventDispatcher through the constructor is needed to test the ViewManager's without having to construct a UIManagerModule.